ZlobinSV/lab_1/main.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main()
+{
+    int i = 0, zam2, n = 0, k = 0, sum[100];
+    string zam1, str, word[100];
+    char a;
+
+    cout << "Enter the string: ";
+
+    getline(cin, str);
+
+    str += " ";
+
+    while (i < str.length() - 1) {
+        while (str[i] != ' ' && str[i] != ',' && str[i] != '.' && str[i] != '!' && str[i] != '?') {
+            word[n] += str[i];
+            i++;
+        }
+        n++;
+        while (str[i] == ' ' || str[i] == ',' || str[i] == '.' || str[i] == '!' || str[i] == '?') {
+            word[n] += str[i];
+            i++;
+        }
+        if (word[n].length()>=2) {
+            if ((word[n][0] == ',' || word[n][0] == '.' || word[n][0] == '!' || word[n][0] == '?') && (word[n][1] == ',' || word[n][1] == '.' || word[n][1] == '!' || word[n][1] == '?')) {
+                cout << "Error! The pinching mark is in the middle of a word or more than one pinning mark appears after / before the word.";
```

Comment:
If input is ` ,, abc` program prints  'Error! The pinching mark is in the middle of a word or more than one pinning mark appears after / before the word.' event though` ,, abc` is a correct input string